### ISSUE TITLE
CPU resources: raw vs scaled

### ISSUE THREAD

**Issue by thockin** [appc/spec#359]
Kubernetes is careful to define CPU in terms of an abstract compute unit.  This leaves room for scaling that compute unit later.  Adding this scaling is VERY hard to do after the fact - assumptions grow very easily about number of threads that can run, concurrency guarantees, etc.

**Comment by jonboulle** [comment]
@thockin glass houses, this is taken _exactly_ from here ;-)

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/resources.md#resource-quantities

**Comment by jonboulle** [comment]
Woops, wrong issue

**Comment by jonboulle** [comment]
This seems much easier for a system like Kubernetes which can abstract this unit and exercise complete control over it. How would we approach this in the spec? Still just define it as abstract units and leave it up to implementations?

**Comment by thockin** [comment]
Yeah, maybe you're right that this is something better left to a higher
level.  How bad is it if a user submits a 4.0 KCU kube job and finds rkt
says "2400 millicores" were assigned?  I think it's not so bad actually...
